docs/bespoke.md

```diff
@@ -0,0 +1,121 @@
+# Bespoke code
+
+Beyond the simple use cases covered by this standard package, most uses of
+MongoDB in Drupal projects appear in enteprise-class bespoke developments. Until
+this version, this usually meant totally custom code, built either straight from
+the legacy `mongo` extension, or on top of the Doctrine ODM for MongoDB,
+suffering from a total lack of integration with the underlying core Drupal CMS.
+
+Starting with 8.x-2.0, such one-off code can be developed on top of the base
+`mongodb` module: unlike earlier releases, 8.x-2.x uses the PHP-standard
+connection methods and options, without deviation, adding only a thin layer of
+Drupal adaptation on top of the standard `mongodb` [extension] and
+[PHP library].
+
+[extension]: http://php.net/mongodb
+[PHP library]: http://mongodb.github.io/mongo-php-library/
+
+
+## Example
+
+The familiar Drupal alias mechanism for databases is available to provide easy,
+string-referenced access to `Client` and `Database` instances through the
+package-provided `ClientFactory` and `DatabaseFactory` services respectively.
+
+Most such code is likely to be service based, so here is an example of a service
```

Comment:
"here is" makes the reader expect something immediately below, doesn't it ? I think it would be better to use something like "the example below", what do you think ?